src/components/common/Actions.vue

```diff
@@ -195,7 +196,11 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Lega
 
       // assume Pay URL is always reachable
       // otherwise user will have to retry payment later
-      window.location.assign(payUrl)
+      if (!paymentCompleted) {
```

Comment:
Which ticket is this for?

Reply:
Ah, this is the left over BCOL PR stuff.
https://github.com/bcgov/bcrs-business-create-ui/pull/114

Reply:
a different one :D I'll revert